### PR TITLE
replaced occurences of google groups email

### PR DESCRIPTION
**Pull request by kaze-droid**
## Context
Issue #3777 
Renamed all occurences of `nusmods<at>googlegroups<dot>com` to `mods<at>nusmods<dot>com`

## Implementation
Replaced all occurences found in:
- README
- scrapers/nus-v2/swagger.yml
- website/src/config/app-config.json
- website/src/views/static/FaqContainer.tsx
- website/static/base/opensearch.xml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ You can reach us through the following channels, listed in order of preference:
 
 - Telegram: https://t.me/nusmods
 - Messenger: https://m.me/nusmods
-- Email: nusmods@googlegroups.com (for security related issues please email mods@nusmods.com instead)
+- Email: mods@nusmods.com (for security related issues please email mods@nusmods.com instead)
```
Comment (kokrui) [inline README.md:31]: This doesn't look right -- now that the default email is `mods@nusmods.com`, there's no need to specify that you should use that email for security-related issues too

**Review by kokrui — CHANGES_REQUESTED** [review]
Mostly LGTM except for the comment below! Make sure you generalize the feedback to other instances where you changed googlegroups -> mods@nusmods

File: README.md (hunk 1 above)
Comment (kaze-droid) [reply README.md]: fixed by removing the back portion in brackets

**Review by zwliew — APPROVED** [review]

**Comment by zwliew** [comment]
Thank you!

**Pull request merged**